modules/core/shared/src/main/scala/lucuma/core/conditions/package.scala

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.conditions
+
+import coulomb.Quantity
+import coulomb.units.accepted.*
+import eu.timepit.refined.*
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.boolean.Not
+import eu.timepit.refined.numeric.Interval as RefinedInterval
+import eu.timepit.refined.numeric.Less
+import lucuma.core.enums.CloudExtinction
+import lucuma.core.enums.Site
+import lucuma.core.enums.SkyBackground
+import lucuma.core.enums.WaterVapor
+import lucuma.core.math.Declination
+import lucuma.core.math.Wavelength
+import lucuma.core.math.erf
+import lucuma.core.util.RefinedNewType
+import lucuma.refined.*
+
+import scala.math.Pi
+import scala.math.abs
+import scala.math.pow
+import scala.math.sin
+
+// Calculations of likelihood of occurrence of observing conditions
```

Comment:
Likelihood over a semester?  On a random night?

Reply:
In many cases it is just a condition has attached a likelihood, however for image quality it also depends on the observing wavelength and airmass, which in turns depends on when and where you're observing